### PR TITLE
Adjust helm chart to use with newer artemis

### PR DESCRIPTION
**Pull request by wojtre**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,8 +9,9 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <helm-maven-plugin.version>2.4.1</helm-maven-plugin.version>
-        <helm.repository.url>https://kubernetes-charts.storage.googleapis.com</helm.repository.url>
+        <helm-maven-plugin.version>2.10.0</helm-maven-plugin.version>
+        <helm.repository.url>https://charts.helm.sh/stable</helm.repository.url>
```
Comment (hho) [inline pom.xml]: I don't think we need this at all, right? The chart has no dependencies to get and isn't published there anyway.
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -24,9 +25,10 @@
                 <configuration>
                     <chartName>artemis</chartName>
                     <chartRepoUrl>${helm.repository.url}</chartRepoUrl>
```
Comment (hho) [inline pom.xml]:
```suggestion
```

**Review by hho — APPROVED** [review]
Only one suggestion

File: pom.xml (hunk 1 above)
Comment (wojtre) [reply pom.xml]: yup, you're right

**Pull request merged**